jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/QueryEditor.kt

```diff
@@ -21,7 +21,7 @@ class QueryEditor internal constructor(private val project: Project) {
     lateinit var searchTerm: JRadioButton
     lateinit var querySearchTerm: JTextField
     lateinit var queryLogGroupsRadioButton: JRadioButton
-    var saveQueryButton: JButton? = null
+    lateinit var saveQueryButton: JButton
```

Comment:
For the next PR, I think you should remove "retrieve saved queries" and add a drop down box that does that